Assignments-003.3.1-Console-Basics-3/Nim-Game-with-Marc/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿//ints for game
+int players = 2;
+int matches = 24;
+int minMatches = 1;
+int maxMatches = 3;
+
+
+Console.WriteLine("Welcome to Nim!");
+
+//Core loop
+while (!IsGameOver())
+{
+    PrintMatches();
+    PlayerDraw();
+    
+    if (matches < 1)
+    {
+        Console.WriteLine("You lose");
+    }
+    else
+    {
+        PrintMatches();
+        AIDraw();
+        
+        if (matches < 1)
+        {
+            Console.WriteLine("AI lose");
+        }
+    }
+}
+
+//--------------------------------------------------
+//Functions for the game
+
+
+//print matches how many matches is left after player and ai has drawn
+void PrintMatches()
+{
+    //Print matches  
+    for (int i = 0; i < matches; i++)

```

Comment:
Here, you can use `Console.WriteLine(new string('|', matches));` it creates a string with as many repetitions of the character as you need :)